src/rgw/motr/gc/gc.cc

```diff
@@ -101,6 +104,11 @@ void *MotrGC::GCWorker::entry() {
           break;
       }
       // unlock the GC queue
+      motr_gc->un_lock_gc_index(my_index);
+    } else {
+      ldpp_dout(dpp, 0) << gc_log_prefix
+          << __func__ << "ERROR: Failed to acquire GC index lock."
```

Comment:
To make admin aware, message can be like  
"ERROR: No GC index is available for locking, all locked ?"

Reply:
I think, message "ERROR: No GC index is available for locking" is sufficient.  OR, how about:
**ERROR: No GC index is available for locking. Skipping GC processing**

Reply:
Go with 
ERROR: No GC index is available for locking. Skipping GC processing